__init__.py

```diff
@@ -185,6 +191,12 @@ def decompile_raw(self):
 
             code = self.decompile(f.name)
 
+            try:
```

Comment:
We are missing the `delete=False` on `with tempfile.NamedTemporaryFile('w+b') as f:`.